src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -36,10 +36,10 @@
   string via (apply str v)."
```

Comment:
Postel's law? I'd rather see this change to accept any other collection, rather than narrowed to just vectors. Unless there's some reason to only take vectors?